control-plane-operator/controllers/hostedcontrolplane/cloud/kubevirt/providerconfig.go

```diff
@@ -48,6 +48,7 @@ func cloudConfig(namespace string) CloudConfig {
 			Enabled:              true,
 			ZoneAndRegionEnabled: false,
 		},
-		Namespace: namespace,
+		Namespace:  namespace,
+		Kubeconfig: kubeconfig,
```

Comment:
it's possible we shouldn't even set this path if the kubeconfig path is empty. I'm not sure how cloud provider kubevirt interprets this if the key exists without a value.

Reply:
in case that the infra cluster is local (i.e. infra cluster = mgmt cluster), the `kubeconfig` key in the `kubevirt-cloud-config` config map is an empty string, and in that case, cloud provider kubevirt uses the local cluster:
https://github.com/kubevirt/cloud-provider-kubevirt/blob/3ee828fd7530b441ca726f72bf34c3e2e88d3a9a/pkg/provider/cloud.go#L105

Reply:
perfect, thanks for looking that up